pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -218,6 +221,15 @@ func (d *ProjectStatusDescriber) Describe(namespace, name string) (string, error
 			printLines(out, indent, 0, describeRCInServiceGroup(f, standaloneRC.RC)...)
 		}
 
+		monopods, err := filterInternalPods(standalonePods)
+		if err != nil {
+			return err
+		}
+		for _, monopod := range monopods {
+			fmt.Fprintln(out)
```

Comment:
Seems like a lot of whitespace between the monopods (nice, btw).  Any reason not to group them all together?  I'll defer to your UI experience, but I don't know that it enhanced my viewing pleasure to have them all separate.